config/packages/easy_admin.yaml

```diff
@@ -62,18 +50,18 @@ easy_admin:
                       type_options:
                           multiple: true
                           expanded: true
-                    - property: UserFullName
-                      label: portal.user.UserFullName
-                    - property: UserDepartment
-                      label: portal.user.UserDepartment
-                    - property: UserAddress
-                      label: portal.user.UserAddress
-                    - property: UserPostalCode
-                      label: portal.user.UserPostalCode
-                    - property: UserCity
-                      label: portal.user.UserCity
-                    - property: UserAccount
-                      label: portal.user.UserAccount
+                    - property: FullName
```

Comment:
The property names should be camelCased. And should the labels.

Reply:
fixed camelCake